docs/guides/reusing_functions.md

```diff
@@ -137,7 +118,6 @@ class DataProcessor:
 
 ## Learn more
 
-For more detailed information about marimo's file format and features, check
-out our [documentation on using your own
-editor](https://docs.marimo.io/guides/editor_features/watching/) or view our
-[file format tutorial](https://marimo.app/?slug=8n55fd).
+For more on marimo's file format, check out our [documentation on using your
+own editor](https://docs.marimo.io/guides/editor_features/watching/) or view
+our [file format tutorial](https://marimo.app/?slug=8n55fd).
```

Comment:
Probably worth converting the tutorials to link.marimo, I'll take the latter half if you grab the first

Reply:
Actually, got them all. prefix `links.marimo.app/tutorial-<>`